stdlib/chiDistributions.hk

```diff
@@ -42,3 +42,25 @@ def nonCentralityParameter(means array(real), stdevs array(prob)):
 
 def ncp(means array(real), stdevs array(prob)):
 	return nonCentralityParameter(means, stdevs)
+
+def F_distribution(n1 nat, n2 nat):
+	X1 <~ standardChiSq(n1)
+	X2 <~ standardChiSq(n2)
+	return (X1*n2)/(X2*n1)
+
+def nonCentralT(n nat, delta prob):
+	U <~ normal(0,1)
+	X <~ standardChi(n)
+	return (U + delta)*sqrt(n)/(X)
+
+def T_distribution(n nat):
+	return nonCentralT(n,0)
+
+def stdCauchy():
+	return T_distribution(1)
+
+def cauchy(a real, alpha prob):
+	X <~ stdCauchy()
+	return a + alpha*X
+
+T_distribution(1)
```

Comment:
Same comment as above - no raw calls.

Also, why are all of these defined in this one file?  Comments would be useful!!!